src/mywxtools.h

```diff
@@ -120,8 +120,7 @@ struct ImagePopup : wxVListBoxComboPopup {
 };
 
 struct ImageDropdown : wxOwnerDrawnComboBox {
-    // FIXME: delete these somewhere
-    Vector<wxBitmap *> bitmaps_display;
+    Vector<unique_ptr<wxBitmap>> bitmaps_display;

```

Comment:
This uses my `Vector` class, which I am not sure interacts correctly with `unique_ptr`.. maybe use `vector` ?

Reply:
So far it works perfectly (tested it).

Reply:
I suggested this may be a source of issues.. lets replace it with `vector`.